.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
Why did you check those files in?

Reply:
As in the extensions those are checked in to maintain uniform styling and inspection profiles across all devs. (i.e. missing final checks)

Reply:
👍🏿